src/components/StoriesInsights/index.js

```diff
@@ -0,0 +1,98 @@
+import RichTypography from "@commons-ui/core/RichTypography";
+import { Grid } from "@material-ui/core";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+import Carousel from "react-multi-carousel";
+
+import Content from "./Content";
+import useStyles from "./useStyles";
+
+import Header from "@/pesayetu/components/Header";
+import Section from "@/pesayetu/components/Section";
+
+import "react-multi-carousel/lib/styles.css";
+
+function StoriesInsights({ overline, title, ctatext, stories, ...props }) {
+  const classes = useStyles(props);
+  const [currentItemIndex, setCurrentItemIndex] = useState(0);
+
+  if (!stories?.length) {
+    return null;
+  }
+
+  const responsive = {
+    desktop: {
+      breakpoint: {
+        max: 3000,
+        min: 1280,
+      },
+      items: 1,
+    },
+    tablet: {
+      breakpoint: { max: 1280, min: 768 },
+      items: 1,
+    },
+    mobile: {
+      breakpoint: { max: 768, min: 0 },
+      items: 1,
+    },
+  };
+  return (
+    <div className={classes.root}>
+      <Section classes={{ root: classes.section }}>
+        <Header overline={overline} className={classes.header}>
+          {title}
+        </Header>
+        <Grid container>
+          <Grid item lg={8} md={12}>
+            <div className={classes.chartContainer}>
+              <RichTypography className={classes.chart}>
+                {stories[currentItemIndex].chart}
+              </RichTypography>
+            </div>
```

Comment:
To improve iframe rendering, I think lets render all all charts here with `display: none` expect the one where `index === currentItemIndex.`

Optional: Add transition similar-ish to how carousel works so that it _feels_ like it's one component.